core/payment/migrations/2020-12-11-123456_event_api/up.sql

```diff
@@ -0,0 +1,15 @@
+DROP TABLE pay_allocation;
+
+CREATE TABLE pay_allocation(
+    id VARCHAR(50) NOT NULL PRIMARY KEY,
+    owner_id VARCHAR(50) NOT NULL,
+    payment_platform VARCHAR(50) NOT NULL,
+    address VARCHAR(50) NOT NULL,
+    total_amount VARCHAR(32) NOT NULL,
+    spent_amount VARCHAR(32) NOT NULL,
+    remaining_amount VARCHAR(32) NOT NULL,
+    `timestamp` DATETIME DEFAULT CURRENT_TIMESTAMP,
```

Comment:
Are you sure the backticks are necessary here? In the initial migration there's `timestamp` used without backticks.

Also, for better precision it should be `DEFAULT(STRFTIME('%Y-%m-%d %H:%M:%f', 'NOW'))`

Reply:
they should all have backticks :P

i'll update it for consistency :)

Reply:
> they should all have backticks :P

But what do you mean by 'should'? They don't have backticks and things seem to work fine.

Reply:
it would be cleaner IMHO, you are correct it works fine